sdk/eventhubs/microsoft-azure-eventhubs-eph/pom.xml

```diff
@@ -14,14 +14,10 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.microsoft.azure</groupId>
   <artifactId>azure-eventhubs-eph</artifactId>
-  <version>3.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-eph;current} -->
+  <version>3.2.3</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-eph;current} -->
 
   <name>Microsoft Azure SDK for Event Hubs Event Processor Host(EPH)</name>
-  <description>EPH is built on top of the Azure Event Hubs Client and provides a number of features not present in that lower layer.${line.separator}
-    This library has been replaced by new Azure SDKs, you can read about them at https://aka.ms/azsdkvalueprop. The latest libraries to interact with the Azure Event Hubs service are:${line.separator}
-    1. https://search.maven.org/artifact/com.azure/azure-messaging-eventhubs${line.separator}
-    2. https://search.maven.org/artifact/com.azure/azure-messaging-eventhubs-checkpointstore-blob${line.separator}
-    It is recommended that you move to the new package.
+  <description>Please note, a newer package com.azure:azure-messaging-eventhubs for Azure Event Hubs is available as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/java/migrate/eh for more details.
```

Comment:
```suggestion
  <description>Please note, a newer package com.azure:azure-messaging-eventhubs at https://search.maven.org/artifact/com.azure/azure-messaging-eventhubs for Azure Event Hubs is available as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/java/migrate/eh for more details.
```

Reply:
Looks good to me. Since this is for the EPH package, it would be good to include the checkpointstore link too.

Reply:
I intentionally left that out as it complicated the messaging because you cant use the checkpoint store in isolation. My hope was to rely on the link to migration guide to fill in the gaps